src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -2651,7 +2651,20 @@ assign(x = ".rs.acCompletionTypes",
          if (string[[whichIndex + 1]] %in% c("list.files",
                                              "list.dirs",
                                              "dir",
-                                             "setwd"))
+                                             "setwd",
+                                             "dir_ls", # fs dir helpers
```

Comment:
Maybe we could just use a plain regular expression for these, e.g.

```
grep("^(?:fs:{2,3})?dir_", string[[whichIndex + 1]])
```

We could also move this to a separate function, e.g. something like `.rs.isDirectoryOnlyCompletionFunction()`?

Reply:
Good idea! Let me know if I placed it in the correct place.